### PR TITLE
Add support for another Aqua format

### LINKED ISSUE #2389: Aqua scanner format no longer working
**Scanner Name**
Aqua.

**Sample File**
@madchap I have a sample file internally from a real report. The gist here is that the old aqua file format uses a top level JSON key called "resources" where each scanned item has a list of vulnerabilities. In the new format (or at least the alternative format?), the top level JSON key is called "cves" and has flattened the structure to be more similar to how the Anchore file format works.

### PR DESCRIPTION
**Pull request by jvz**
@madchap 

Supersedes #2708.

This fixes #2389

### CODE REVIEW

**Comment by madchap** [comment]
@jvz this one is still trying to merge to `master` and not `dev`?

**Comment by madchap** [comment]
I could change it manually here without trouble, since everything is so fresh due to the latest release and given the scope of the PR. I relaunched a Travis, since it failed somehow on generating the nginx image.

File: dojo/tools/aqua/parser.py [hunk 1]
```diff
@@ -100,4 +95,41 @@ def get_item(resource, vuln, test):
...
+    solution = item.get('solution')
+    fix_version = item.get('fix_version')
+    if solution:
+        mitigation = solution
+    elif fix_version:
+        mitigation = f'Upgrade to {fix_version}'
+    else:
+        mitigation = 'No known mitigation'
+
+    return Finding(title=f'{cve}: {file_path}',
```
Comment (madchap) [inline dojo/tools/aqua/parser.py:114]: It seems that `name` would contain what would be considered the `file_path` or `component_name` in DefectDojo, e.g. for java components.

**Comment by valentijnscholten** [comment]
Turns out this PR introduced some flake8 errors, fixed in #2726 2726

**Pull request merged**